backends/rapidpro/backend_test.go

```diff
@@ -65,6 +65,13 @@ func (ts *BackendTestSuite) SetupSuite() {
 		log.Fatalf("unable to start backend for testing: %v", err)
 	}
 
+	// read our schema sql
+	sqlSchema, err := ioutil.ReadFile("schema.sql")
```

Comment:
FYI the postgres action actually has an option by where you provide a directory of SQL scripts to be run when database is being initialized.. but couldn't get it to work when I was converting mailroom..

Reply:
I had the same issue as well